javascript/javascript1/week2/task4.js

```diff
@@ -0,0 +1,39 @@
+console.log('Task4 - Student manager')
+
+const class07Students = [];
+function addStudentToClass(studentName) {
+  
+  if (studentName == '') {
```

Comment:
- I recommend using [trim](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/Trim) to include long empty strings